Makefile.am

```diff
@@ -215,6 +215,15 @@ _usnic_files = \
 
 if HAVE_VERBS
 _usnic_files += prov/usnic/src/usdf_fake_ibv.c
+
+# If the verbs or usnic providers are being built as a DL, then we
+# need to link libibverbs into the usnic provider
+if HAVE_VERBS_DL
+_usnic_libibverbs = -libverbs
+endif
+if HAVE_USNIC_DL
+_usnic_libibverbs = -libverbs
+endif
 endif
```

Comment:
Ahhh that is what I was missing: I thought it was always requiring linking against libibverbs. But it appears that it uses verbs.h always, but only links/calls into libibverbs when the verbs provider is also built (although it can safely always register the fake driver because it depends on libibverbs). So never mind I think this is doing the right thing.